templates/statefulset-patroni.yaml

```diff
@@ -27,6 +27,8 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         env:
+        - name: PGVERSION
+          value: "10"
```

Comment:
By default, spilo 11 will use Postgres 11, we need to specify our PG version here (we are using 10)